miri-script/src/util.rs

```diff
@@ -146,3 +146,46 @@ impl MiriEnv {
         Ok(())
     }
 }
+
+/// Receives an iterator of files.
+/// Will format each file with the miri rustfmt config.
+/// Does not follow module relationships.
+pub fn format_files(
```

Comment:
Please put this into the `impl MiriEnv`, then you also need to pass fewer parameters.